be/src/olap/column_vector.h

```diff
@@ -150,7 +150,7 @@ class ScalarColumnVectorBatch : public ColumnVectorBatch {
 class ArrayNullColumnVectorBatch : public ColumnVectorBatch {
 public:
     explicit ArrayNullColumnVectorBatch(ColumnVectorBatch* array)
-            : ColumnVectorBatch(get_scalar_type_info(FieldType::OLAP_FIELD_TYPE_TINYINT), false),
+            : ColumnVectorBatch(get_scalar_type_info<FieldType::OLAP_FIELD_TYPE_TINYINT>(), false),
```

Comment:
we can also delete prefix`FieldType::` here, and other places.

Reply:
ok